app/src/test/java/com/example/mac/chartr/activities/PostTripActivityTest.java

```diff
@@ -39,6 +33,23 @@ public void setup() {
     }
 
 
+    @Test
+    public void testInitPickers() {
+        PostTripActivity activity = Robolectric.buildActivity(PostTripActivity.class)
+                .create().get();
+
+
+        EditText departureDate = activity.findViewById(R.id.editTextDepartureDate);
```

Comment:
Make these global variables, we only want to findViewById for each object once per file if possible